.github/workflows/stellar-dominion-build-and-push.yml

```diff
@@ -37,7 +35,7 @@ jobs:
           push: true
           tags: totocorpsoftwareinc/stellar-dominion:${{ needs.extract-service-tag.outputs.version }}
   persist-service-tag:
-    if: ${{ github.actor != 'github-actions[bot]' && github.event_name != 'push' || github.event.pusher.name != 'github-actions[bot]' }}
+    if: ${{ github.ref == 'refs/heads/master' }}
```

Comment:
This is the main change (here and in other workflows): instead of checking for the bot and not the branch we directly check the branch name here. We don't want to even call the persist tag workflow if the branch is not `master`. This solves the problem we had :+1: